python/tvm/meta_schedule/tune.py

```diff
@@ -596,8 +600,23 @@ def tune_relay(
     target = default_config.target(target)
     # pylint: enable=protected-access,
     # parse the tuning contexts
+
+    if executor is None:
+        executor = relay.backend.Executor("graph")
+
+    if "link-params" in executor.attrs:
+        link_params = executor.attrs["link-params"]
+    else:
+        link_params = False
+
     with Profiler.timeit("TaskExtraction"):
-        extracted_tasks = extract_task_from_relay(mod, target, params)
+        pass_config = {
+            "relay.FuseOps.link_params": link_params,
+            "relay.backend.use_meta_schedule": True,
+            "relay.backend.tir_converter": "default",
+        }
```

Comment:
See https://github.com/apache/tvm/blob/370abe69d24519a5453cead846d328a1c378957f/python/tvm/meta_schedule/relay_integration.py#L87-L91 for why this change is necessary. We only need to pass `relay.FuseOps.link_params` config, others are for compatibility with the existing code.